crosshair/libimpl/builtinslib_test.py

```diff
@@ -455,6 +462,30 @@ def f(a: str) -> int:
 
         self.assertEqual(*check_ok(f))
 
+    def test_rindex_fail(self) -> None:
+        def f(a: str) -> int:
+            """ post: _ != 2 """
+            try:
+                return a.rindex("abc")
+            except ValueError:
+                return 0
+
+        self.assertEqual(*check_fail(f))
+
+    def test_rindex_err(self) -> None:
+        def f(a: str) -> int:
+            """ post: _ == -1 """
```

Comment:
Consider `post: True` to be extra sure that we don't accidentally find a counterexample before we hit the exception.

Reply:
Done in latest commit.